test/specs/abConfiguration.spec.js

```diff
@@ -13,130 +27,130 @@ describe('ABConfiguration', function() {
             }
         });
 
-        this.visitor = new Visitor({
+        testContext.visitor = new Visitor({
             id: 'visitor_id',
             assignments: []
         });
-        this.logErrorStub = sandbox.stub(this.visitor, 'logError');
+        testContext.visitor.logError = jest.fn();
     });
 
-    it('requires a splitName', function() {
+    test('requires a splitName', () => {
```

Comment:
since we're using jest, can't we keep using the bdd style here with describe/it? i prefer that style.

Reply:
lemme see if i can get this working

Reply:
done!